### ISSUE TITLE
Add game option for textual order input.

### ISSUE THREAD

**Issue by zond** [zond/diplicity#136]
This would do a few things:

- Turn off the order options structure being sent to the client.
- Turn off error response codes for invalid orders.
- Refrain from sending orders for unresolved phases to clients.

This would force games to not validate orders, and give players leeway for breaking orders.

@tttppp, @JorenC what do you guys think?

**Comment by zond** [comment]
Like

kie m ber

or

kie move ber

Case insensitive, I assume

**Comment by JorenC** [comment]
Hahahaha.

That's not what I mean.
I mean that we need to have some sort of "add order" interface; I would
still want to have one rule per order (clearly separated) and some sort of
"add order" thing...

**Comment by JorenC** [comment]
"delete order"

**Comment by zond** [comment]
Yeah - OR we could do it "like face to face" and give a text field, where we just split the input by line?

I mean - the point here is to give people enough rope to hang themselves, right?

**Comment by zond** [comment]
But, we still don't want to let people write messages and use the order view as a chat.

If we store the order input as text, and resolve them as "try to parse orders line by line", then we might want to filter out everything that isn't an order type, a unit type, or a province from the text before storing/displaying it?

**Comment by JorenC** [comment]
Yes, I think that makes sense.

I would just make sure you have them enter it line by line, so any errors
in order #1 don't have any impact on order #2.

**Comment by zond** [comment]
Oh, we can solve that even if they enter everything in a text field. But in the end it's a UX question, so I'll let you decide.
